src/main/java/com/regnosys/rosetta/common/serialisation/json/reportdata/ExpectedResult.java

```diff
@@ -1,4 +1,4 @@
-package com.regnosys.rosetta.common.serialisation.reportdata;
+package com.regnosys.rosetta.common.serialisation.json.reportdata;
```

Comment:
This change has nothing to do with this change. If we want to move it, I think we should do this in an isolated way so it does not impact the ISO 20022 delivery.